tests/components/orderentry/PatientDashboard.test.jsx

```diff
@@ -1,10 +0,0 @@
-import React from 'react';
-
-import PatientDashboard from '../../../app/js/components/orderEntry/PatientDashboard';
-
```

Comment:
Was removing this part of the work for this ticket?

Reply:
I believe so, from my understanding with the team, I was to remove unwanted and duplicated CSS from our files. The main aim although was putting all the external css into the `opemrs-owa-orderentry.css` file.